pipelines/ApPipe.yaml

```diff
@@ -35,6 +35,11 @@ tasks:
     config:
       connections.coaddName: parameters.coaddName
       doSkySources: True
+      python: |
+        import lsst.meas.extensions.shapeHSM
+        config.measurement.slots.psfShape = "ext_shapeHSM_HsmPsfMoments"
+        config.measurement.slots.shape = "ext_shapeHSM_HsmSourceMoments"
+        config.measurement.plugins.names |= ["ext_shapeHSM_HsmSourceMoments", "ext_shapeHSM_HsmPsfMoments"]
```

Comment:
Why can't we make these changes in `DetectAndMeasureConfig` directly? It's a diffim-specific task, and we probably want its default config to be appropriate for measuring on difference images.

Reply:
Agreed; I've opened a PR for ip_diffim to do just that: https://github.com/lsst/ip_diffim/pull/247